### PR TITLE
Update CODEOWNERS to reflect latest owners

### PR DESCRIPTION
**Pull request by yorinasub17**
NOTE: I originally had the codeowners officially only be me, but have a team section so I know who to route requests to, but in the new world with contractors and team owners, it no longer makes sense to adopt that model so I am officially setting all the current SMEs as code owners.

### CODE REVIEW

**Comment by autero1** [comment]
Ah, ok. I updated the `CODEOWNERS` with `github-watcher.py` and that just seemed to append to the end of the file. This applies to other repos / PRs, as well but you seem to have issued new PRs for those.

**Review by zackproser — APPROVED** [review]
LGTM

**Comment by yorinasub17** [comment]
Thanks for review! Merging now!

@autero1 Ah gotcha! Yea I noticed this after I merged all of them and hastily went through opening PRs 😸

**Pull request merged**
